lib/home/balance/balance_filters_modal.dart

```diff
@@ -14,9 +20,17 @@ class BalanceFiltersModal extends StatefulWidget {
 }
 
 class _BalanceFiltersModalState extends State<BalanceFiltersModal> {
-  String selectedValue = 'week';
-  bool isButtonEnabled = false;
-  bool isButtonLoading = false;
+  final balanceCubit = Modular.get<BalanceScreenCubit>();
+  final filtersCubit = Modular.get<BalanceScreenFiltersCubit>();
+  bool buttonEnabled = false;
+  late FetchMovementsFilters state;
+
+  @override
+  void initState() {
+    state = filtersCubit.state;
+    super.initState();
+  }
```

Comment:
In this case it won't affect anything, but it is always interesting to place the `super.initState();` first.

This article explains it https://medium.com/flutter-senior/call-super-initstate-first-call-super-dispose-last-9de47fdb932d